charts/mailu/templates/_services.tpl

```diff
@@ -175,6 +175,7 @@ Service fqdn (within cluster) can be retrieved with `mailu.SERVICE.serviceFqdn`
 {{- if .Values.ingress.enabled -}}
     {{- $enabledPorts = append $enabledPorts "80" -}}
     {{- $enabledPorts = append $enabledPorts "443" -}}
+    {{- $enabledPorts = append $enabledPorts "4190" -}}
```

Comment:
You need to get 587 into this list for submission to work
otherwise https://github.com/Mailu/Mailu/blob/master/core/nginx/dovecot/proxy.conf#L162 fails

`mailu.enabledPorts` Feeds into `envvars-configmap.yaml` Line 117 `PORTS` that is used in https://github.com/Mailu/Mailu/blob/master/core/base/libs/socrate/socrate/system.py#L144-L147 which would set `PORT_587`  which the above proxy.conf line uses